datastore/longhorn.go

```diff
@@ -193,10 +193,53 @@ func (s *DataStore) ValidateSetting(name, value string) (err error) {
 		if err := types.ValidateCPUReservationValues(guaranteedEngineManagerCPU.Value, guaranteedReplicaManagerCPU.Value); err != nil {
 			return err
 		}
+	case types.SettingNameStorageNetwork:
+		if err := types.ValidateStorageNetwork(value); err != nil {
+			return err
+		}
```

Comment:
Redundant. This is already validated in the above invocation `types.ValidateSetting(name, value)`.

Reply:
Removed

Reply:
Removed.